doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -218,13 +218,17 @@ numbers (see :ref:`datatypes`).
 
 .. index::
    single: Set (sort)
+   single: SProp
    single: Prop
    single: Type
 
 Sorts
 -----
 
-There are three sorts :g:`Set`, :g:`Prop` and :g:`Type`.
+There are four sorts :g:`SProp`, :g:`Prop`, :g:`Set`  and :g:`Type`.
+
+-  :g:`SProp`_is the universe of *definitionally irrelevant
```

Comment:
```suggestion
-  :g:`SProp` is the universe of *definitionally irrelevant
```

Reply:
This is what made the refman build fail BTW.

Reply:
Oh I see, emacs displays the non breaking space (used below for Prop etc) as an underscore so I typed _ assuming sphinx would translate it.